src/app/app.component.html

```diff
@@ -29,6 +29,17 @@
         </span>
     </span>
     <span class="toolbar-spacer"></span>
+
+    <span>
+        <a
+            *ngIf="(network$ | async).faucet && !isMobile()"
+            mat-button
+            class="faucet-button"
+            (click)="openFaucet()"
+        >
+            {{ 'Get ' + (network$ | async).shortName | uppercase }}
```

Comment:
Would it be easy to replace the text with an icon? 
For example this [gift-card](https://material.io/tools/icons/?search=card&icon=card_giftcard&style=baseline) material icon. You can use an angular material [icon-button](https://material.angular.io/components/button/examples)

Also it would be nice to have a matTooltip with something like `Opens faucet` for the button.

Reply:
I just found that if it takes some space to display your ether, then the transaction counter is cut on mobile: 
![mobilefaucet](https://user-images.githubusercontent.com/34456797/60651639-f3ff3980-9e46-11e9-9bc2-1b74c0b3b452.png)
A small icon might be a bit better, but I think we need to rethink the whole mobile header completely or don't place the faucet button in the header on mobile.